src/main/java/tds/exam/results/messaging/ExamCompletedMessageListener.java

```diff
@@ -70,7 +71,11 @@ public void handleMessage(final String examId) {
     private void processMessage(final String examId) {
         // Once this ERT message has been received, the exam status is "submitted"
         final UUID id = UUID.fromString(examId);
-        examResultsService.findAndSendExamResults(id);
-        examService.updateStatus(id, ExamStatusCode.STATUS_SUBMITTED);
+        TDSReport report = examResultsService.findAndSendExamResults(id);
+
+        //Expired Exams can have TRT's created.  However, those statuses do not get updated.
+        if(!report.getOpportunity().getStatus().equals(ExamStatusCode.STATUS_EXPIRED)) {
```

Comment:
I left the check on expired but maybe we only set it to "submitted" if it is "completed"?   Anyhow, this fixes the bug where the ERT would retry 3 times and then fail.